app/uk/gov/hmrc/agentservicesaccount/models/desiDetails/DesiDetails.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentservicesaccount.models.desiDetails
+
+import play.api.libs.json.{Json, OFormat}
+import uk.gov.hmrc.agentservicesaccount.models.AgencyDetails
+
+case class DesiDetails (
```

Comment:
```suggestion
case class DesignatoryDetails (
```
Just for context as whoever maintains this going forward may not understand